network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -74,24 +83,38 @@ def get_query_output(query_answer, only_explanation=False, add_explanation=False
     def get_query_type():
         return QueryType.SingleConfigQuery
 
+    def policy_title(self, policy):
```

Comment:
Because the one defined in NetworkConfigQuery overrides the base one and calls policy.full_name with self.config.name. The base one is good for TwoNetworkConfigsQuery too (which does not have self.config)